src/components/Menu/Menu/__stories__/interactions.js

```diff
@@ -0,0 +1,22 @@
+import {
+  getByRole,
+  getByText,
+  clickElement,
+  hoverElement,
+  testFunctionWrapper,
+  delay
+} from "../../../../__tests__/interactions-helper";
+
+export const showSubMenus = testFunctionWrapper(async canvas => {
+  const menuElement = await getByRole(canvas, "menu");
+  // Click the sub menu
+  let optionToSelect = getByText(menuElement, "With Sub menu");
+  await hoverElement(optionToSelect);
+  // Click the sub sub menu
+  optionToSelect = getByText(menuElement, "Sub Sub menu");
+  await hoverElement(optionToSelect);
+  // validate showing sub sub item
+  await delay(1);
```

Comment:
is there a way to "wait for element to be visible"? Some testing frameworks allow waiting for a condition to be truthy before continuing..
If not, consider adding a comment here for why you added the delay :)